juju/juju
# Add missing CMR CLI filter support

## Description of change

The CMR find command can now filter using interface and offer name.
There's a couple of filter options not yet supported (role, endpoint name app description) - these can be add if needed at some stage.

Also clean up the cmd help text.

## QA steps

Deploy and offer an application
Ensure that find -- offer and --interface works.
